Jenkinsfile

```diff
@@ -1232,6 +1234,9 @@ pipeline {
                         not {
                             environment name: 'PLATFORM', value: 'sles11'
                         }
+                        not {
```

Comment:
Question, you can't add 2 lines under one `not { }`?

Reply:
Unfortunately you can't. I've tried that in the past and it was a syntax error.

Reply:
Ok thanks, yea I was not sure.